cmake/checks/check_02_compiler_features.cmake

```diff
@@ -452,7 +465,12 @@ IF(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 ELSEIF(CMAKE_CXX_COMPILER_ID MATCHES "Intel")
   ADD_FLAGS(CMAKE_REQUIRED_FLAGS "-diag-error warn")
 ENDIF()
-ADD_FLAGS(CMAKE_REQUIRED_FLAGS "-Werror -fuse-ld=gold")
+IF(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
+  ADD_FLAGS(CMAKE_REQUIRED_FLAGS "/WX")
+ELSE()
+  ADD_FLAGS(CMAKE_REQUIRED_FLAGS "-Werror")
+ENDIF()
+ADD_FLAGS(CMAKE_REQUIRED_FLAGS "-fuse-ld=gold")
```

Comment:
I think this can't really happen with MSVC, right?

Reply:
Yes, `-fuse-ld=` won't be recognized. I disabled that feature check for `MSVC` now.